### PR TITLE
#15: Generated APIs should import models from generated model.ts (closes #15)

### LINKED ISSUE #15: Generated APIs should import models from generated model.ts

### PR DESCRIPTION
**Pull request by veej**
Closes #15

## Test Plan

> These results can be screenshots, query results, or even just pointers to unit tests that successfully passed.}

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.}

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -232,11 +232,13 @@ function getRoute(route: Route, isReadonly: boolean): string {
 const getRoutesPrelude = `// DO NOT EDIT MANUALLY - metarpheus-generated
 import axios from 'axios'
 import { pathReporterFailure } from 'io-ts/lib/reporters/default'
+import * as t from 'io-ts'
+import * as m from 'model-ts'
```
Comment (giogonzo) [inline src/index.ts]: I'd make this more self contained by importing from '. /model'

**Review by giogonzo — CHANGES_REQUESTED** [review]
Looks good, but see inline comment. I also think we should try this on a real project later today

File: src/index.ts (hunk 1 above)
Comment (veej) [reply src/index.ts]: mmm... I'm not sure I understand. Do you mean `./model-ts`?

**Review by giogonzo — APPROVED** [review]
Code looks 👍 

What are we missing in order to use this on A-PRO?
I remember you removed maybe a couple not-yet supported routes involving path params (not sure..)

As a first step, in order to mark this as "done", I'd like to backport on a real project and fix any regressions
(to be clear: I mean backporting this on master, even if benefits aren't yet "visible" for the dev passing through the still untyped `container` libs)

**Comment by veej** [comment]
@giogonzo yep, path params are still not supported. I had to modify few routes on A-PRO (`configurations/workcell`, `configurations/workcellGroup`) temporarily removing path params from scala controllers.

Apart from this, a PR on `scriptoni` is needed to upgrade `metarpheus-io-ts`, enable runtime models export and generate APIs with prelude.

**Pull request merged**